pkg/store/didanchor/store_test.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package didanchor
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/trustbloc/orb/pkg/store/mocks"
+)
+
+//nolint:lll
+//go:generate counterfeiter -o ./../mocks/store.gen.go --fake-name Store github.com/hyperledger/aries-framework-go/spi/storage.Store
+//go:generate counterfeiter -o ./../mocks/provider.gen.go --fake-name Provider github.com/hyperledger/aries-framework-go/spi/storage.Provider
+
+func TestNew(t *testing.T) {
+	t.Run("success", func(t *testing.T) {
+		s, err := New(&mocks.Provider{})
+		require.NoError(t, err)
+		require.NotNil(t, s)
+	})
+
+	t.Run("error - open store fails", func(t *testing.T) {
+		provider := &mocks.Provider{}
+		provider.OpenStoreReturns(nil, fmt.Errorf("open store error"))
+
+		s, err := New(provider)
+		require.Error(t, err)
+		require.Contains(t, err.Error(), "failed to open did anchor store: open store error")
+		require.Nil(t, s)
+	})
+}
+
+func TestStore_Put(t *testing.T) {
+	t.Run("success", func(t *testing.T) {
+		provider := &mocks.Provider{}
```

Comment:
For success cases, I suggest using the real in-memory store implementation (found [here](https://github.com/hyperledger/aries-framework-go/blob/main/component/storageutil/mem/mem.go)). If storage has to be updated later, I've found it easier when dealing with real storage implementations. It also makes the unit tests act closer to the real thing.

Reply:
Sure, I will use in-memory store for success scenario.

Reply:
Fixed